### DISCUSSION TITLE
重启后需要清空磁盘数据才可以正常使用

### DISCUSSION THREAD

**Discussion by bestRiven** [milvus-io/milvus#16748] [Q&A and General discussion]
milvus2.0.2单机版和集群版我这边都是每次挂了，都需要清空磁盘数据etcd和minio才可以正常使用，否则就是看起来是起来了但是使用一会就又不行了，这个是什么原因呢？

**Comment by xiaofan-luan** [comment]
一般是两种原因，一是磁盘满了，一般都是search往mq写了很多数据，二是某个地方卡住了，已知的问题是compaction删除了还没load的数据导致卡住，其他的case需要提供报错信息和完整日志排查。
如果遇到类似情况，可以先检查磁盘util，如果没有满，可以提供下日志开个issue交给社区同学看看

**Reply by bestRiven** [reply]
那可能是第二种情况，目前磁盘肯定是没满，这种情况有什么解决方案吗？

**Reply by houxingxing** [reply]
您好， 我也遇到了同样的问题，重启后要半个小时之后查询才正常。 请问删掉volumes会把写入的表数据删掉吗？
